src/makeRobot.js

```diff
@@ -38,7 +38,64 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const evacuationX = 1400;
+  const evacuationY = 500;
+  const startCoords = {
+    x: 0,
+    y: 0,
+  };
+
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: startCoords,
+
+    get info() {
+      return `name: ${this.name}, `
+        + `chip version: ${this.version}, `
+        + `wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    getValidDistance(distanceMoved = 1) {
+      return distanceMoved < 0 ? 0 : distanceMoved;
+    },
+
+    goForward(distanceY) {
```

Comment:
Would be better make this arg name the same in all relevant methods
```suggestion
    goForward(steps) {
```